### PR TITLE
[Database metrics] Add operations we want metrics for 

### PR DESCRIPTION
**Pull request by chinandrew**
Summary  of changes
- Add the functions we are interested in obtaining metrics for. These will be passed into a later method that takes a method and captures metrics while they run. Also no tests since they're all one liners and make calls to docker or the api, so mocking them out wouldn't tell us much.
- Add init file since I forgot it

### CODE REVIEW

File: src/database_metrics/operations.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        Data source name
+    file_pattern: str
+        Filename of pattern to match
+
+    Returns
+    -------
+    Bytestring of Docker log, either STDOUT or STDERR
+    """
+    return client.containers.run(
+        image=client.images.get("delphi_python"),
```
Comment (dfarrow0) [inline src/database_metrics/operations.py:50]: in the future, if this runs in automation, you'll likely want to build `delphi_python` prior to running it. (no action needed atm)

File: src/database_metrics/operations.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    params: dict
+        Query parameters to send. List of parameters can be found here:
+        https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html#constructing-api-queries
+
+    Returns
+    -------
+    Requests Response object.
+    """
+    req = requests.get("http://localhost:10080/epidata/api.php",
+                       auth=("user", "pass"),
```
Comment (dfarrow0) [inline src/database_metrics/operations.py:23]: what are these creds used for? the api shouldn't require http auth.

**Review by dfarrow0 — COMMENTED** [review]
nice docstrings!

File: src/database_metrics/operations.py (hunk 2 above)
Comment (chinandrew) [reply src/database_metrics/operations.py:23]: oops didn't even realize this, was using them for some other queries and just threw them in. removed.

File: src/database_metrics/operations.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                       params=params)
+    return req
+
+
+def load_data(client: DockerClient, source: str, file_pattern: str) -> bytes:
+    """
+    Ingest data into epidata database using the Python docker image.
+
+    Copies files from common_full/covidcast/receiving/`source`/`file_pattern`, which should be
+    copied into via the DockerFile, to the common/covidcast/receiving/`source`/ folder
```
Comment (chinandrew) [inline src/database_metrics/operations.py:32]:
```suggestion
    copied to the image via the DockerFile, to the common/covidcast/receiving/`source`/ folder
```

File: src/database_metrics/operations.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
+"""Operations for which metrics are captured for."""
```
Comment (krivard) [inline src/database_metrics/operations.py:1]: This edit may not yield a good docstring, but it's less bad:

```suggestion
"""We can collect metrics on the data transformation procedures defined in this file."""
```

* this repo is named "operations" and it's a different "operations" than the "operations" meant here. Avoid overloading if possible
* for which X are Y for? no

**Review by krivard — APPROVED** [review]
Optional file rename but otherwise good

File: src/database_metrics/operations.py (hunk 4 above)
Comment (chinandrew) [reply src/database_metrics/operations.py:1]: Good call, will rename. `actions.py`?

Comment (krivard) [reply src/database_metrics/operations.py]: sgtm

Comment (chinandrew) [reply src/database_metrics/operations.py]: changed

**Pull request merged**